terraform/init.tf

```
@@ -0,0 +1,43 @@
+terraform {
+  required_version = ">= 1.2.2"
+  backend "azurerm" {}
+  required_providers {
+    azurerm = "~> 3.11.0"
```

if you're updating it may as well use the latest
```suggestion
    azurerm = "3.44.1"
```

---

sure